internal/compose/sync.go

```diff
@@ -0,0 +1,270 @@
+package compose
```

Comment:
Although the goal of this code is to fetch compose files from Beats' integrations, I'd not put this sync code under the `compose` package, which is managing the deployment of services using Docker Compose. I think it could fit more under the `state` or even `sanitizer` packages, but not having a strong opinion. What other package would you use?